### ISSUE TITLE
Compute Loss over test set and per class

### ISSUE THREAD

**Issue by jsalbert** [triagemd/keras-eval#13]
Equivalent of  evaluate_generator of https://keras.io/models/model/
